README.md

```diff
@@ -1,2 +1,31 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+**Pseudocode for Fallout 4 hacking game**
+
+	guesses = 4 # Number of guesses used for loop
+	guesses_left = 4 # Used to track guesses left 
+	word = random word from database # The word to be checked agaisnt
+	word_length = number of characters in chosen word # Length of word
+
+	print 'The password is ' word_length ' characters long, you have ' guesses ' guesses to get it right!'
+
+	for i in range(0, guesses): # Loops as many times as guesses allowed or until broken by the loop
+		if i == 4 # If 4 guesses have been done print failure and break loop
+			print 'Out of guesses, the auorities have been contacted and will arive shortly'
+			break
+		print 'What is your guess?'
+		get inputed guess
+		right = how many characters of the inputed string match against word including position # checks inputed guess with the word to see how many characters match type and position.
```

Comment:
The worksheet asked you to produce pseudocode to calculate this. Be sure to read specifications carefully in future!

Reply:
Also, `right` is not a good variable name as it could be confused with the direction -- `correct` would better, `numCorrectLetters` better still